src/transaction/transaction.dto.ts

```diff
@@ -9,6 +10,19 @@ export class TransactionDTO {
     amount: number;
 }
 
+export class BaseTransactionDTO {
+    @ApiProperty({ required: true })
+    amount: number;
+}
+
+export class TransferTransactionDTO extends BaseTransactionDTO {
+    @ApiProperty({ required: true })
+    senderId: string;
+
+    @ApiProperty({ required: true })
+    receiverId: string;
+}
+
 export class AdminTransferTransactionDTO {
```

Comment:
Can you remove `AdminTransferTransactionDTO`. And also refactor all code that use this DTO since it is a duplicate of `TransferTransactionDTO`. *use `refactor-> rename`. I'm not sure if VS code have this kind of refactor to replace all

If I remember correctly there is only one use of `AdminTransferTransactionDTO` in `admin.controller.ts` in case you want to refactor it manually